website/src/app/[lang]/[region]/v2/(home)/(sections)/monthly-income.tsx

```diff
@@ -10,7 +10,10 @@ export async function MonthlyIncome({ lang, region }: DefaultParams) {
 	});
 
 	return (
-		<BaseContainer backgroundColor="bg-foreground-dark" className="py-3">
+		// <BaseContainer backgroundColor="bg-foreground-dark" className="py-3">
+		<BaseContainer className="theme-blue-v2 py-3">
+			{' '}
+			{/* Is there a way to use theme-blue-v2 to set background blue? */}
```

Comment:
This comment is not needed anymore, right? You are setting the theme now...

Reply:
This wasn't working as expected on my local system or on the deployed one either, is it on yours?

<img width="1512" alt="Screenshot 2024-07-21 at 20 09 40" src="https://github.com/user-attachments/assets/97287298-53d0-4809-b5e3-8a124049a7b5">

Ideally it should be screen-wide but that is not happening for some reason